pkg/csi/service/common/constants.go

```diff
@@ -359,4 +359,6 @@ const (
 	// TopologyPreferentialDatastores is the feature gate for preferential
 	// datastore deployment in topology aware environments.
 	TopologyPreferentialDatastores = "topology-preferential-datastores"
+	// 255 CSI volumes per Node support
```

Comment:
Can we start the docstring with `TwoFiftyFiveVolumesPerNode ...`

Reply:
done. and renamed this gate to `max-pvscsi-targets-per-vm`